### DISCUSSION TITLE
Top sources - forever "No data yet"

### DISCUSSION THREAD

**Discussion by smiklosovic** [plausible/analytics#802] [Self-Hosted Support] [answered]
Hi,

we are running this in our staging environment for Apache Cassandra project and all looks good but I am not sure why Top Sources are empty.

I was trying to do something like

`https://cassandra.staged.apache.org?utm_source=Newsletter`

(you got the idea), but nothing is adding numbers into dashboard. What are we doing wrong?

https://plausible.cassandra.apache.org/cassandra.staged.apache.org

**Comment by metmarkosaric** [comment, marked as answer]
hi @smiklosovic! Plausible is already showing your sources but all of them have been direct/none until now so they don't show in the main report. See https://plausible.cassandra.apache.org/cassandra.staged.apache.org/sources?period=day

The referral sources are counted only when they start a new session on your site so chances are you visited your site directly on the same day before you also tried to visit it using the UTM tagged link which means we won't include it in referral sources.

Pretty cool to see a project like this using Plausible! hope we can get a mention somewhere on your site as it would help us spread the word to even more projects/sites which would be good in getting more of them to remove GA!

**Reply by smiklosovic** [reply]
Thanks it works ... I connected via VPN to some other country and tried that link and entry is there nicely ....

Yeah we are evaluating this and if we happen to indeed use it we might do something about your idea for sure. Looks good so far!

Thanks again!

**Reply by metmarkosaric** [reply]
nice and thanks for the update! enjoy Plausible!

**Reply by smiklosovic** [reply]
@metmarkosaric its live for a while to see how it goes ... https://plausible.cassandra.apache.org/cassandra.apache.org?period=day

**Reply by metmarkosaric** [reply]
looking nice! looking forward to hearing what you think!
